apps/coscrad-frontend/src/configurable-front-matter/data/configurable-content-schema.ts

```diff
@@ -39,6 +39,7 @@ export type ConfigurableContent<T extends CategorizableType = CategorizableType>
     termOfTheDayConfig?: Record<string, string>;
     notFoundMessage: string;
     loadingMessage: string;
+    theme: Record<string, string>;
```

Comment:
This is not good. You have no type safety whatsoever on the keys \ values of this object. What you want to do is to build up a partial type from the MUI theme type either using Partial, or maybe better- Pick.